resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -36,41 +36,50 @@
     language: bash
     program: |
 
-      # make folders with data
+      FILE_PATH=()
+      SAMPLE_NAMES=()
+      BUILD=()
+      SPECIES=()
+      EXP_TYPE=()
+
       {% for d in data %}
-        {% set species = d.species or 'other_data' %}
         {% for field in fields %}
           {% if d[field].file %}
-            mkdir -p {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/
-            ln -s --backup=numbered {{ d[field].file }} {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/{{ d[field].file|basename }}
+            mkdir -p {{ d|sample_slug }}/{{ field }}/
+            ln -s --backup=numbered {{ d[field].file }} {{ d|sample_slug }}/{{ field }}/{{ d[field].file|basename }}
             {% for ref in d[field].refs %}
-              ln -s --backup=numbered {{ ref }} {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/{{ ref|basename }}
+              ln -s --backup=numbered {{ ref }} {{ d|sample_slug }}/{{ field }}/{{ ref|basename }}
             {% endfor %}
           {% endif %}
           {% for file in d[field] %}
             {% if file.file %}
-              mkdir -p {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/
-              ln -s --backup=numbered {{ file.file }} {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/{{ file.file|basename }}
+              mkdir -p {{ d|sample_slug }}/{{ field }}/
+              ln -s --backup=numbered {{ file.file }} {{ d|sample_slug }}/{{ field }}/{{ file.file|basename }}
               {% for ref in file.refs %}
-                ln -s --backup=numbered {{ ref }} {{ species }}/{{ d.build }}/{{ d|sample_slug }}/{{ field }}/{{ ref|basename }}
+                ln -s --backup=numbered {{ ref }} {{ d|sample_slug }}/{{ field }}/{{ ref|basename }}
               {% endfor %}
             {% endif %}
           {% endfor %}
         {% endfor %}
 
-        # select data for IGV
+        #  select data for IGV from the same Species and Build
         {% if (d|type).startswith('data:alignment:bam') %}
-          mv {{ species }}/{{ d.build }}/{{ d|sample_slug }}/bai/{{ d.bai.file|basename }} {{ species }}/{{ d.build }}/{{d|sample_slug}}/bam/{{ d.bai.file|basename }}
-          rm -r {{ species }}/{{ d.build }}/{{d|sample_slug}}/bai
-          echo {{ species }}/{{ d.build }}/{{ d|sample_slug }}/bam/{{ d.bam.file|basename }} >> {{ species }}_{{ d.build }}_temp_igv.txt
+          mv {{ d|sample_slug }}/bai/{{ d.bai.file|basename }} {{d|sample_slug}}/bam/{{ d.bai.file|basename }}
+          rm -r {{d|sample_slug}}/bai
+          echo {{ d|sample_slug }}/bam/{{ d.bam.file|basename }} >> {{ d.species }}_{{ d.build }}_temp_igv.txt
 
         {% elif (d|type).startswith('data:variants:vcf') %}
-          echo {{ species }}/{{ d.build }}/{{ d|sample_slug }}/vcf/{{ d.vcf.file|basename }} >> {{ species }}_{{ d.build }}_temp_igv.txt
+          echo {{ d|sample_slug }}/vcf/{{ d.vcf.file|basename }} >> {{ d.species }}_{{ d.build }}_temp_igv.txt
```

Comment:
check what it is not IGV map in this case: https://qa2.genialis.com/swiftbio/search?_b=&_s=062c8acd-321e-4876-a1c4-7821d07cfbae